Wrappers/Python/ccpi/viewer/utils/colormaps.py

```diff
@@ -1108,7 +1108,7 @@ def relu(x, xmin, xmax, scaling=1):
         if val < xmin or val > xmax:
             out.append(0)
         else:
-            out.append((val - xmin) / dx)
+            out.append((val - xmin)*scaling / dx)
```

Comment:
just for clarity, could you change the position of scaling?

```python
out.append( scaling * ((val-xmin) / dx) )
```

Reply:
done